### DISCUSSION TITLE
Can not display space size of cloud storage (Mountain Duck)

### DISCUSSION THREAD

**Discussion by andymachs** [iterate-ch/cyberduck#16358] [Q&A] [answered]
Mountain Duck can not display space size of cloud storages. also can not set the space size manually.

**Comment by dkocher** [comment, marked as answer]
Usually cloud storage has not quota applied for the storage capacity. We translate quota when available to disk size information. You can find information in our documentation [^1] about quota support.

[^1]: https://docs.mountainduck.io/mountainduck/connect/#quota-support

**Comment by andymachs** [comment]
I connect WebDAV via Mountain Duck, it always display 7.99 EB, it is wrong,
if Mountain Duck can not display correctly, please support setting manually.
What's more, Mountain Duck can not display free space of cloud storages.

**Reply by dkocher** [reply]
We will take this as a feature request.

**Reply by tatze96** [reply]
Regarding the cloud storages:
To which cloud storage do you refer?

Most of the cloud storages are mounted with virtual top-level folders that don't have a quota, hence why exabyte values (*Online* mode) or, respectively, the available disk space within the synchronization cache location (*Smart Synchronization* mode) are displayed. The quota should be displayed correctly once a path other than `/` is mounted.

**Reply by andymachs** [reply]
Alipan Drive

https://www.alipan.com

**Reply by PublicBenemy** [reply]
> We will take this as a feature request.

This would be really awesome!

**Comment by Jakeverse** [comment]
I'm using Koofr 1TB plan, mounted drive shows 100gb total space and 63gb free which totally wrong info. Why not use api based mounting like airlivedrive or winfsp based native like mount used by rclone? It's so much problem for normal user like us as we just want right information on our drive.

**Reply by dkocher** [reply]
We do not support connecting through Kofr that acts like a proxy and may put its own storage limits.

**Reply by Jakeverse** [reply]
Sad to hear, I like mountain duck for cryptomator inbuilt and bought license but it natively don't support my main cloud. Thanks for information. 😢
